rest_framework_json_api/renderers.py

```diff
@@ -170,6 +170,7 @@ def extract_relationships(fields, resource, resource_instance):
                                 'meta': {
                                     'count': len(resource.get(field_name))
```

Comment:
This is the line that throws the error.  Since your patch is below this line, it won't prevent the error from being thrown.